README.md

```diff
@@ -68,6 +68,32 @@ that looks like this:
 }
 ```
 
+## Enrich tracking events
+
+You can add application-specific fields to tracking events by calling
+`Holmes.addTrackingEnricher(enricherFunc)`. The client library then calls those
+functions whenever you call `Holmes.track()` and passes the tracking event
+object as argument. The `enricherFunc` can add additional data to this event.
+
+To make sure that Holmes is fully loaded when you call `addTrackingEnricher()`,
+you should wait for the DOM event named "holmesloaded" on the `window` object
+before registering enrichers.
+
+In the following example two fields are added to every tracking event.
+`applicationVersion` will be set to `1.2.3` and `applicationDate` to the current
+date:
+
+```javascript
+window.addEventListener('holmesloaded', function() {
```

Comment:
I would document this event earlier in the README, perhaps in a separate section since it has nothing to do with enriching events but rather with the general usage of the client lib.

Reply:
Done